iree/modules/check/CMakeLists.txt

```diff
@@ -57,6 +57,7 @@ iree_cc_binary(
     iree::base::status
     iree::base::target_platform
     iree::base::tracing
+    iree::hal::dylib::dylib_driver_module
```

Comment:
missing a corresponding change to `iree/modules/check/BUILD`?

Reply:
Its in IREE_DRIVER_MODULES list in `bazel_to_cmake_converter.py` which is updated in this PR. CI didn't run it !!?

Reply:
Huh, weird. Let's re-run the GitHub actions checks to see if it's a transient error. This does look correct on closer inspection...

Reply:
Still shows the same diff. @GMNGeoffrey any ideas?

Reply:
weird... Let me try fiddling with it locally

Reply:
Ok the issue is that the workflow is checking out the merge commit for the action which somehow does not have your bazel_to_cmake change. https://github.com/google/iree/commit/89dd7e42866c334f89b57def2605b274d9d16ca6 I don't understand why though. My recommendation would be to rebase and force push and see if that fixes it.

Reply:
Rebased, it works now.